docs/pages/advanced/advanced-failover.md

```diff
@@ -219,7 +219,7 @@ In addition to the properties specified above depending on the chosen mode, ther
 
 | Property | Notes |
 | failover.tcp.host | Optional property, only used with Direct TCP mode that specifies the host name or IP address of the current Interlok instance.  If not specified we will try to determine the local IP address. |
-| failover.slave.position | if you wish to preconfigure the slave position in the hierarchy then define this, otherwise one will be assigned |
+| failover.secondary.position | if you wish to preconfigure the secondary position in the hierarchy then define this, otherwise one will be assigned |
```

Comment:
- Check that this renders properly since in the currnt live docs; it's not rendered as a table.
- You need to raise a ticket so that we change the associated "properties" in the underlying code!